app/serializers/nodeinfo/serializer.rb

```diff
@@ -37,6 +37,10 @@ def open_registrations
     Setting.registrations_mode != 'none' && !Rails.configuration.x.single_user_mode
   end
 
+  def metadata
+    Array([])
```

Comment:
`Array([])` is not idiomatic ruby code. Please use `[]` instead, which is identical.

Reply:
Sorry! This was literally my first time reading/writing Ruby.